rosette/run.sh

```diff
@@ -1,6 +1,9 @@
+#! /usr/bin/env bash
+
 # Increase stack limit so that Rosette can load all files
 ulimit -s unlimited
 
 # Execute Rosette instance
-export ESS_SYSDIR=rbl/rosette
-./build.out/src/rosette --quiet --interactive-repl --boot-dir=rbl/rosette --boot=boot.rbl $@
+export ESS_SYSDIR="$(readlink -f $(pwd)/result/lib/rosette)"
```

Comment:
Should we use pwd -P for this?
See "Perhaps use pwd -P to prevent the path from containing symlinks?"
from @nashef on https://github.com/rchain/rchain/pull/457

Reply:
In this case, `result` is actually a symlink to a build artifact create in `/nix/store`, so we actually don't want to use `pwd -P`